docs/source/conf.py

```diff
@@ -215,8 +215,6 @@
     "undoc-members": True,
     # Don't document things with a leading underscore.
     "private-members": True,
-    # Include special members.
```

Comment:
decided seeing every single __init__ is obnoxious

Reply:
If you want some inits, you can specify them at the file level.